utils/po_stat.sh

```diff
@@ -0,0 +1,13 @@
+cd ../po/
```

Comment:
Please fix this one. I can mark the other two about the subprocess module to be Ignored.